pkg/controller/utils/list_test.go

```diff
@@ -0,0 +1,52 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+package utils
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestContainsString(t *testing.T) {
```

Comment:
Not something introduced in this PR, but the function under test does the same as the one you already tested in `func Test_StringsContains`. Should we get rid of one of them?

Reply:
right, i noticed that. we could put them in a `common` file, perhaps in a separate PR